src/sections/disaggregation.rst

```diff
@@ -730,413 +299,102 @@ items:
 
-In model settings, hazard groups will be specified in a similar way to the existing groups, in data settings. However, the 
-model provider will be able to specify the fields differently for hazard than damage.
+There is also a difference between cases in how disaggregated risks are grouped for the purposes of correlating hazard and damage in the ground up loss calculation. 
 
-**Example 1:**
+The model provider can control this through the model settings json and more guidance can be found in the correlation section.
 
-``Model_settings.json``
+In case it is not specified, the default setting in Oasis is to fully correlate the subrisks for campus sites in case 3 (same correlation group id is assigned) and to make the subrisks for aggregate data independent in case 2 (different correlation group_id per subrisk).
 
-.. code-block:: JSON
+**Usage settings**
 
-    "data_settings": {
-    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber", site_id] "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber", disagg_id]
-    },
 |
 
-In this example, using the new site_id field, hazard group will be generated differently for the disaggregated locations for 
-the IsAggregate=1 case, but the same for the disaggregated locations for the IsAggregate=0 case, which may be assumed to be 
-geographically proximate, and therefore receive 100% correlation in hazard intensity. However damage will be correlated at 
-the disaggregated location level in both cases.
+It is necessary to use the gulmc calculation module to use number of risks and financial terms disaggregation as they are not supported in the gulpy calculation module.
 
-|
-
-items:
-
-.. csv-table::
-    :header: "item_id", "coverage_id", "areaperil_id", "vulnerability_id", "damage_group_id", "hazard_group_id"
-
-    "1", "1", "100001", "101", "1", "1"
-    "2", "2", "100001", "103", "1", "1"
-    "3", "3", "100001", "101", "2", "2"
-    "4", "4", "100001", "103", "2", "2"
-    "5", "5", "100002", "101", "3", "3"
-    "6", "6", "100002", "103", "3", "3"
-    "7", "7", "100002", "101", "4", "3"
-    "8", "8", "100002", "103", "4", "3"
-    "9", "9", "100002", "101", "5", "3"
-    "10", "10", "100002", "103", "5", "3"
+In oasislmf json settings, this is how to specify the calculation module options;
 
 |
 
-**Example 2:**
-
-``Model_settings.json``
+``oasislmf.json``
 
 .. code-block:: JSON
 
-    "data_settings": {
-    "hazard_group_fields": ["PortNumber", "AccNumber", "LocNumber", ] "damage_group_fields": ["PortNumber", "AccNumber", "LocNumber",     ]
-    }
-
-|
-
-The model provider could instead use the disagg_id field for hazard grouping, to assign independent hazard groups to both 
-IsAggregate cases, in cases of a highly localized peril where 100% hazard correlation cannot be assumed, even for a campus.
-
+    "modelpy": true,
```

Comment:
I think this should be `"modelpy": false,`  since its not used with gulmc?

Reply:
Yes have tested removal of both modelpy: false and gulpy: false in oasislmf settings when using gulmc=true, and it works and produces same results, so I think they are not necessary.